mailpoet/lib/AdminPages/Pages/Help.php

```diff
@@ -117,4 +127,33 @@ private function getLatestActionSchedulerActionDate(string $hook, string $status
     }
     return null;
   }
+
+  private function buildTaskData(ScheduledTaskEntity $task): array {
```

Comment:
I'd be curious to hear your reasoning for moving buildTaskData() to Help instead of to ScheduledTasksRepository. I don't see a problem with this path, I'm just wondering.

Reply:
1) The `buildTaskData`  is only used in Help.php ( probably not even usable elsewhere since it's very much coupled with what's expected in the template of this page)
2) Repository seems to be a good place to interact with DB and fetch data from it, but this method is simply changing the structure of the data ( other the side-effect `$queue = $this->sendingQueuesRepository->findOneBy(['task' => $task]);` that's there and is simply copied around)